src/coreclr/jit/valuenumfuncs.h

```diff
@@ -116,6 +116,7 @@ ValueNumFuncDef(GetsharedNongcstaticBase, 2, false, true, true)
 ValueNumFuncDef(GetsharedGcstaticBaseNoctor, 1, false, true, true)
 ValueNumFuncDef(GetsharedNongcstaticBaseNoctor, 1, false, true, true)
 ValueNumFuncDef(ReadyToRunStaticBase, 1, false, true, true)
+ValueNumFuncDef(ReadyToRunCctorTrigger, 1, false, true, true)
```

Comment:
I don't think you should end up needing this. Now that this is `TYP_VOID` it is going to get void value number.

Reply:
I still think this (and the `ReadyToRunStaticBase` above it) can be removed.